files/nrpe/check_openstack.py

```diff
@@ -191,28 +195,32 @@ def provide_keystone(self):
   def provide_keystone_v3(self):
     return self.keystone_v3_cred
 
-class OSVolumeCheck(cinder.Client):
+def keystone_session_v3(options):
```

Comment:
I'm confused. I thought at some point someone said that the support for sessions is deprecated. Was that just Keystone v2 sessions?

Reply:
Short answer I don't know, I think it is different kind of session keystoneauth1 vs keystoneclient.

check_keystone gives a warning in Newton that session is deprecated but check_openstack does not.